webpack.config.js

```diff
@@ -87,6 +87,16 @@ module.exports = {
 
   optimization: {
     noEmitOnErrors: true,
-    concatenateModules: true
+    concatenateModules: true,
+    splitChunks: {
+      chunks: 'all',
+      name: 'resources/js/vendor/biotope-element',
+      cacheGroups: {
+        vendors: {
+          test: /[\\/]@biotope[\\/]element[\\/]/,
+          priority: -10
```

Comment:
Q: `-10` seems like a magic number. Is there a meaning to it, or is it just meant to be a low number?

Reply:
Bump

Reply:
@SheepFromHeaven can you answer the open question so we can merge?

Reply:
It's just a question, I'm ok merging without any changes.
The intent was just to understand the value's meaning 😄

Reply:
@tiagomapmarques this gives the user the possibility to add custom cache groups himself which bundle biotope/element. Without a low priority, our cachegroup would always be prefered.

Reply:
=> so just a low number